transforms-caching-experiment/remote-cache-experiment-transforms.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Note:
+# This is a script to run the transforms remote caching experiments. You need to update it to use your server url, gradle version (8.9+) and set up the access key with remote cache write permission (you can also export it in the terminal if you wish to share the script with teammates). When executing, you can also supply a different task to run this with, it defaults to help. It will run 3 builds:
```

Comment:
> you can also export it in the terminal if you wish to share the script with teammates

Can you just copy `~/.gradle/develocity/keys.properties` to `$homeDir/develocity/keys.properties`?

Reply:
Sure, but most folks don't have the remote cache write permission for local development, so I chose to keep it separate.

Reply:
You have this line in the README:

> Having the key set in the environment is expected - note that remote build cache write is requiered. You can set the key in the environment by running `export DEVELOCITY_ACCESS_KEY=<develocity-url>=<your-access-key>`, either in the script, or in the terminal before running the script.

Which sounds like instructions for running locally. All you would need to do is add an additional `cp` and you could remove the whole requirement about the access key needing to be set in the environment or hardcoded in the script and it doesn't matter where you run the experiment. CI or local, it should just work, as long as you had the right permissions to begin with.

Reply:
I reworded the section and added copying the keys to the script